### DISCUSSION TITLE
Should Gatus user-agent use the current gatus version?

### DISCUSSION THREAD

**Discussion by heitorPB** [TwiN/gatus#549] [Q&A] [answered]
In https://github.com/TwiN/gatus/blob/master/core/endpoint.go#L35 there is this line:

```go
	GatusUserAgent = "Gatus/1.0"
```

But the current Gatus version, as in the latest tag os of now, is 5.5.1.

So, should that change based on the latest tag/version?

As a suggestion, I can make a simple Bash script `prepare-release.sh` that:
- modifies the `core/endpoint.go` file with the new specified version
- commits it with a message of `Release version x.y.z`
- creates an annotated tag

After that you can review the change and `git push` (or have that in the script as well).

Later on this could be extended to have a section of a changelog in the tag as well :)

**Comment by TwiN** [comment, marked as answer]
I don't think it's worth the effort.

Ultimately, Gatus' User-Agent header is only there to give a way for people to differentiate requests coming from Gatus.

**Reply by heitorPB** [reply]
It appears that it is the standard to use the version of the product: https://en.wikipedia.org/wiki/User-Agent_header

Prometheus did it in a cool way:
https://github.com/prometheus/prometheus/pull/2447/files

But yeah, this is minor stuff.
